src/etc/inc/plugins.inc.d/unbound.inc

```
@@ -117,6 +119,13 @@ function unbound_generate_config()
         mwexecf('/sbin/mount -t devfs devfs %s', '/var/unbound/dev');
     }
 
+    $python_dir = "/usr/local/lib/{$pythonv}";
+    $chroot_python_dir = "/var/unbound{$python_dir}";
+
+    if (mwexec("/sbin/mount | grep {$chroot_python_dir}")) {
```

```suggestion
    if (mwexec("/sbin/mount | grep {$chroot_python_dir}", true)) {
```

Because we can ignore error since we handle it here